pom.xml

```diff
@@ -22,7 +22,7 @@
       (for scm, site-distribution, etc -->
     <project_name>fcrepo</project_name>
     <project_organization>fcrepo</project_organization>
-    <fcrepo-storage-ocfl.version>6.3.0-SNAPSHOT</fcrepo-storage-ocfl.version>
+    <fcrepo-storage-ocfl.version>6.3.0</fcrepo-storage-ocfl.version>
```

Comment:
Shouldn't this be `6.4.0-SNAPSHOT` now?

Reply:
@dbernstein when you have a moment (so we don't forget to get these changes back in main), can you check this? I feel like it should be altered.

Reply:
sure - I'll fix that.